package.json

```diff
@@ -746,7 +763,7 @@
         {
           "command": "vscode-couchbase.runSearch",
           "category": "Couchbase",
-          "when": "(editorLangId == cbs.json || resourceFilename =~ /.cbs.json$/)  && !isKVCluster",
+          "when": "editorLangId == json && resourceFilename =~ /\\.cbs\\.json$/  && !isKVCluster && isSearchEnabled",
```

Comment:
why "||" to "&&"?

Reply:
This is because I updated the languageId to be json it self and not cbs.json, due to which we need the conditional 
"editorLangId == json && resourceFilename =~ /\\.cbs\\.json$/ to be true, otherwise it will trigger for all json files which is not what we want.